gocourse08/collartask/animal/animal.go

```diff
@@ -0,0 +1,52 @@
+package animal
+
+import "math/rand/v2"
+
+type AnimalType string
+
+const (
+	Bear  AnimalType = "bear"
+	Tiger AnimalType = "tiger"
+	Lion  AnimalType = "lion"
+)
+
+type Animal interface {
+	Typify() string
+	RecordBreathing(breathingData float64)
+	RecordSound(soundData float64)
+}
+
+type DataAnimal[T float64] struct {
+	AnimalType  string
+	Pulse       int
+	Temperature float64
+	Breathing   []T
+	Sounding    []T
```

Comment:
Це ж слайси, значить закінчення `s`:
```suggestion
	Breaths   []T
	Sounds    []T
```